src/Morphir/Visual/ViewList.elm

```diff
@@ -67,20 +118,33 @@ view config viewValue itemType items =
                                 )
                     }
 
-            _ ->
-                viewAsList config viewValue items
+            Type.Reference _ _ _ ->
+                -- display small lists of simple values horizontally
+                if List.length items < 10 then
```

Comment:
I'm not sure if this is going to capture only "simple" items. It captures everything that is not a record, which could be complex custom types or even collections. If we cannot decide if these are small values I think it's better to just go with the safe solution and show everything using the default layout.